src/main/java/io/phasetwo/keycloak/magic/auth/MagicLinkAuthenticator.java

```diff
@@ -83,12 +85,14 @@ public void action(AuthenticationFlowContext context) {
     if (user == null
         || MagicLink.trimToNull(user.getEmail()) == null
         || !MagicLink.isValidEmail(user.getEmail())) {
-      context.getEvent()
-              .detail(AbstractUsernameFormAuthenticator.ATTEMPTED_USERNAME, email)
-              .event(EventType.LOGIN_ERROR).error(Errors.INVALID_EMAIL);
       context
-              .getAuthenticationSession()
-              .setAuthNote(AbstractUsernameFormAuthenticator.ATTEMPTED_USERNAME, email);
+          .getEvent()
+          .detail(AbstractUsernameFormAuthenticator.ATTEMPTED_USERNAME, email)
+          .event(EventType.LOGIN_ERROR)
+          .error(Errors.INVALID_EMAIL);
+      context
+          .getAuthenticationSession()
+          .setAuthNote(AbstractUsernameFormAuthenticator.ATTEMPTED_USERNAME, email);
```

Comment:
Is there a change here? I can't tell.

Reply:
I didn’t change that; it was probably the formatter.